<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>curPicForLabeling_MainWindow</class>
 <widget class="QMainWindow" name="curPicForLabeling_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>保存</string>
    </property>
    <addaction name="actionendLabeling"/>
    <addaction name="actionremoveTheLast"/>
   </widget>
   <widget class="QMenu" name="saveImediately">
    <property name="title">
     <string>马上保存</string>
    </property>
    <addaction name="actionsavePicOnly"/>
    <addaction name="actionsaveLabelsOnly"/>
    <addaction name="actionsaveAll"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>更改形状</string>
    </property>
    <addaction name="actionchangeToAnyFourPoly"/>
    <addaction name="actionchangeToRectMode"/>
    <addaction name="actionchangeToFivePoly"/>
    <addaction name="actionchangeAndActivateSelectMode"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>更改画笔</string>
    </property>
    <addaction name="actionchangeColor"/>
    <addaction name="actionchangeWidth"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>管理辅助标记</string>
    </property>
    <addaction name="actionopenTheLabelHelper"/>
    <addaction name="actioncloseTheLabelHelper"/>
    <addaction name="actionchangeHelperPenColor"/>
    <addaction name="actionchangeHelperPenWidth"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="saveImediately"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionendLabeling">
   <property name="text">
    <string>结束标记</string>
   </property>
  </action>
  <action name="actionsavePicOnly">
   <property name="text">
    <string>只保存图片</string>
   </property>
  </action>
  <action name="actionsaveLabelsOnly">
   <property name="text">
    <string>只保存标签</string>
   </property>
   <property name="iconText">
    <string>只保存标签</string>
   </property>
  </action>
  <action name="actionsaveAll">
   <property name="text">
    <string>保存所有</string>
   </property>
  </action>
  <action name="actionchangeToAnyFourPoly">
   <property name="text">
    <string>切换到任意四边形标注</string>
   </property>
  </action>
  <action name="actionchangeToRectMode">
   <property name="text">
    <string>切换到标准矩形标注</string>
   </property>
  </action>
  <action name="actionchangeToFivePoly">
   <property name="text">
    <string>切换到任意五边形</string>
   </property>
  </action>
  <action name="actionchangeAndActivateSelectMode">
   <property name="text">
    <string>切换到自定义</string>
   </property>
  </action>
  <action name="actionchangeColor">
   <property name="text">
    <string>更换颜色</string>
   </property>
  </action>
  <action name="actionchangeWidth">
   <property name="text">
    <string>更换粗度</string>
   </property>
  </action>
  <action name="actionremoveTheLast">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionopenTheLabelHelper">
   <property name="text">
    <string>打开辅助标记</string>
   </property>
  </action>
  <action name="actioncloseTheLabelHelper">
   <property name="text">
    <string>关闭辅助标记</string>
   </property>
  </action>
  <action name="actionchangeHelperPenColor">
   <property name="text">
    <string>更改辅助画笔画笔颜色</string>
   </property>
  </action>
  <action name="actionchangeHelperPenWidth">
   <property name="text">
    <string>改变辅助画笔标记笔宽</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
